app/routers/event.py

```diff
@@ -27,6 +28,41 @@ async def eventview(request: Request, id: int):
                                       {"request": request, "event_id": id})
 
 
+def get_event_by_id(db: Session, event_id: int) -> Event:
+    """Select event by id"""
+    return db.query(Event).filter(Event.id == event_id).first()
+
+
+def validate_dates(start_date: datetime, end_date: datetime) -> bool:
+    """Check if the start date is earlier than the end date"""
+    return start_date < end_date
+
+
+def update_event(event_id: int, event_dict: Dict, db: Session
```

Comment:
Maybe just `events`? We usually try not to use [Hungarian Notation](https://en.wikipedia.org/wiki/Hungarian_notation) when it's possible :)

Reply:
*event